index.js

```diff
@@ -32,6 +36,13 @@
 // console.log("%c----------", "color: red") 
 
 
+function drinkWater(current_thirst_level){
+  console.log("Man I sure am thirsty")
+  current_thirst_level -= 1
+  console.log("Ahh that hits the spot")
+  return current_thirst_level
+}
```

Comment:
Very good! Good job using camelCase for the function name. It's also convention to use camelCase instead of snake_case for variable names in JavaScript.
```suggestion
function drinkWater(currentThirstLevel){
  console.log("Man I sure am thirsty")
  currentThirstLevel -= 1
  console.log("Ahh that hits the spot")
  return currentThirstLevel
}
```